vault-toolkit/vault-install-plugin.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This script registers vault-plugin-secrets-github plugin.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Validations and defaults
+: "${VAULT_CACERT:?Need to set VAULT_CACERT}"
+local_addr="${VAULT_LOCAL_ADDR:-"https://127.0.0.1:8200"}"
+
+# Wait until vault is ready for registration
+until curl -Ss -f --cacert "${VAULT_CACERT}" "${local_addr}/v1/sys/init" | jq -e '.initialized == true or .initialized == false' >/dev/null 2>&1; do
+  echo "vault not ready, sleeping for 3 seconds"
+  sleep 3
+done
+
+if curl -Ss -f --cacert "${VAULT_CACERT}" "${local_addr}/v1/sys/init" | jq -e '.initialized == false' >/dev/null 2>&1; then
```

Comment:
Should this be `until`?

Currently it will sleep 3s and move on regardless of the output.